src/components/Console.jsx

```diff
@@ -49,13 +49,19 @@ export default function Console({
         className="label console__label"
         onClick={partial(onToggleVisible, currentProjectKey)}
       >
-        Console
-        <span className="console__chevron u__icon">{chevron}</span>
+        <div>
+          Console
+          <span className="console__chevron u__icon">{chevron}</span>
+        </div>
         <span
```

Comment:
I would make this a `div` as well since it’s a flex item (I think practically it doesn’t matter, `<div>` just seems more semantic)